src/Datadog.Trace/Tagging/HttpTagsKeysMapping.cs

```diff
@@ -0,0 +1,11 @@
+namespace Datadog.Trace.Tagging
+{
+    internal struct HttpTagsKeysMapping
```

Comment:
After the `OtelHttpTags` refactor, is this used anywhere?

Reply:
no. nice catch. thanks 👍 totally missed it when I was reviewing my own PR 😉

Reply:
Removed